### DISCUSSION TITLE
Mining Turtle Help (Begineer)

### DISCUSSION THREAD

**Discussion by MLGMagicHoodini** [cc-tweaked/CC-Tweaked#1896] [Q&A] [answered]
I'm recently trying use CC:Tweaked, and turtles, I got a turtle to dig a 5x5 Hole, and all, but some issues happen and I don't know how to fix it, the main one is if I go too far away, and come back it will stop, when I rejoin the world, he's not working, and the way I fix it is pick him up, put him at starting position near the chest, and do the "excavate 5" to start the task, and wait for him get down the hole and continue his job, any way to improve or fix this?

**Comment by zyxkad** [comment]
Related #535

Turtle will not work when the chunk is not loading, there are three ways to solve the problem:
1. Keep a player stays with the turtle, or ensure the turtle stays in any player's view distance
2. Use chunkloader from other mods. If you like to make the turtle keep moving you can use [AP](https://github.com/IntelligenceModding/AdvancedPeripherals)'s [Chunky turtle](https://advancedperipherals.netlify.app/turtles/chunky_turtle/)
3. Make turtle save and continues its work after restart

Notice way 1 and way 2 will not keep turtle work after while server restarting

**Reply by MLGMagicHoodini** [reply]
Hmm the chunkloader I can get behind, any idea how to make it always return to a starting position? Or how to start with server restart? I'm new to this and don't know LUA (or how to use Turtle console)

**Reply by zyxkad** [reply, marked as answer]
> any idea how to make it always return to a starting position?

Yes, [setup a gps system](https://tweaked.cc/guide/gps_setup.html) and make turtle go to [specific gps](https://tweaked.cc/module/gps.html#v:locate) everytime it restarts.
However you need to learn how to write Lua by yourself, for a free online manual you can check [this](https://www.lua.org/manual/5.2/)
